MantidPlot/src/Mantid/MantidUI.cpp

```diff
@@ -1377,6 +1361,15 @@ Table *MantidUI::createDetectorTable(
       }
       const QString isMonitorDisplay = isMonitor ? "yes" : "no";
       colValues << QVariant(specNo) << QVariant(detIds);
+      if (isScanning) {
+        std::set<int> timeIndexSet;
+        for (auto def : spectrumInfo.spectrumDefinition(wsIndex)) {
+          timeIndexSet.insert(int(def.second));
+        }
+
+        QString timeIndexes = createTruncatedList(timeIndexSet);
+        colValues << QVariant(timeIndexes);
+      }
```

Comment:
So if there is more than one detector for the spectrum you would get duplicate time indices and cannot associate them to detectors? Probably this is not a common case, but just to make sure this is not overlooked. Should it simply be ignored? Or generate a warning? Throw?